### PR TITLE
Added TTGO target

### PR DESCRIPTION
**Pull request by wvarty**

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -126,8 +170,13 @@ void loop()
   {
     sampleAd8318();
 
-    drawPlot();
-
+    if (InputWithinBounds()) {
+      drawPlot();
+    }
+    else {
+      drawOverpowerWarning();
```
Comment (JyeSmith) [inline src/main.cpp]: How about still drawing the plot and showing mW/db, but display over power warning in the middle of the screen.  It is still useful to see the info for debugging purposes.

File: src/main.cpp [hunk 2]
```diff
@@ -23,27 +28,47 @@ uint8_t meanSampledBm = 0;
...
-  uint32_t tempReading = 0;
-  for (int i = 0; i < SCREEN_WIDTH; i++)
-  {
-    tempReading = analogRead(AD8318_PIN);
-    tempReading += analogRead(AD8318_PIN);
-    tempReading += analogRead(AD8318_PIN);
-    samples[i] = tempReading / 3;
-  }
+  #ifdef ARDUINO_TARGET
+    for (int i = 0; i < SCREEN_WIDTH; i++)
```
Comment (JyeSmith) [inline src/main.cpp]: Hmmm dat i++

Comment (wvarty) [reply src/main.cpp]: Missed one.... was trying to keep your existing code the same ;)

File: src/main.cpp (hunk 1 above)
Comment (wvarty) [reply src/main.cpp]: Can do I suppose

**Comment by wvarty** [comment]
@JyeSmith I changed the overpower warning, but now my TTGO refuses to flash for some unknown reason, so its stuck on the current build forever :|

Can you test the warning? Or if it's too much effort just decline the PR, either way I dont mind.

**Pull request merged**